terraform/aks/workspace_variables/review_aks.tfvars.json

```diff
@@ -7,5 +7,9 @@
   "namespace": "bat-qa",
   "deploy_azure_backing_services": false,
   "db_sslmode": "prefer",
-  "webapp_startup_command": "bundle exec rails db:prepare && bundle exec rails server -b 0.0.0.0"
+  "webapp_startup_command": "bundle exec rails db:prepare && bundle exec rails server -b 0.0.0.0",
+  "webapp_memory_max": "3Gi",
+  "worker_memory_max": "2Gi",
+  "secondary_worker_memory_max": "2Gi",
+  "clock_worker_memory_max": "1Gi"
```

Comment:
bearing in mind that the review apps are dev environments - are we able to test using less memory?